### PR TITLE
Deprecate lsp-factory.js

### PR DESCRIPTION
**Pull request by Hugoo**
There are still a few outdated occurrences to lsp-factory in our guides.

@mustafademiray this will be something to check and update

### CODE REVIEW

File: docs/learn/expert-guides/universal-profile/create-profile.md [hunk 1]
```diff
@@ -20,12 +20,6 @@ To easiest way to create a Universal Profile is through the Universal Profile Br
 - install the 🖥️ [Universal Profile Browser Extension](/install-up-browser-extension)
 - and ✨ [create a Universal Profile](https://my.universalprofile.cloud)
 
-However, if you want to create a profile programmatically you should use the ⚒️[lsp-factory.js](../../../tools/lsp-factoryjs/deployment/universal-profile.md). You can install it in your project using:
```
Comment (CJ42) [inline docs/learn/expert-guides/universal-profile/create-profile.md:23]: Suggest moving this into a callout and linking to the LSP23 guide. (Need to change the link in my code example.

```
:::success Alternative guide

For developers looking for a guide to create a profile programmatically, see our guide [**Deploy a UP + its Key Manager using the LSP23 Factory**](./link/to/learn/expert-guides/deploy-UP-with-LSP23).

:::
```

File: docs/learn/expert-guides/universal-profile/create-profile.md [hunk 2]
```diff
@@ -66,23 +60,6 @@ For more details, see [LSP6 - Key Manager Standard](../../../standards/universal
 
 ![Universal Profile: overview of deployed contracts](../img/universal-profile-overview.jpeg)
 
-The [lsp-factory.js](../../../tools/lsp-factoryjs/getting-started.md) library will help us quickly deploy and set up a Universal Profile with just a few lines of code.
```
Comment (CJ42) [inline docs/learn/expert-guides/universal-profile/create-profile.md:69]: Hmmm it is good to remove this big section, but what is then the relevancy and the goal of this guide then? To still deploy the UP + Key Manager manually without using LSP23?

File: docs/tools/lsp-factoryjs/getting-started.md [hunk 3]
```diff
@@ -4,151 +4,12 @@ sidebar_position: 1.1
...
-
-:::note
-If you already have LSP3 data uploaded, then you can pass an IPFS URL:
-
-```javascript
-const myLSP3MetaData = 'ipfs://QmPzUfdKhY6vfcTNDnitwKnnpm5GqjYSmw9todNVmi4bqy';
-```
+- [Deploying Universal Profiles with LSP23 Factory](../../learn/expert-guides/deploy-up-with-lsp23/)
+- [Deploying tokens and NFTs](../../learn/digital-assets/smart-contract-developers/getting-started)
+- [`lsp-utils`](../lsp-utils/getting-started.md) library
```
Comment (CJ42) [inline docs/tools/lsp-factoryjs/getting-started.md:13]: Suggest adding small sentence for what this library is good for:

```suggestion
- [`lsp-utils`](../lsp-utils/getting-started.md) library to easily set profile and asset metadata.
```

File: docusaurus.config.js [hunk 4]
```diff
@@ -145,6 +145,39 @@ export default {
             from: '/tools/partners',
             to: '/tools/integrations',
           },
+          // lsp-factory deprecation
```
Comment (CJ42) [inline docusaurus.config.js:148]: Very nice and well anticipated 👌🏻⭐️

File: sidebars.js [hunk 5]
```diff
@@ -151,45 +151,45 @@ module.exports = {
...
           type: 'autogenerated',
-          dirName: 'tools/lsp-smart-contracts',
+          dirName: 'tools/lsp-utils',
         },
       ],
     },
     {
       type: 'category',
-      label: '📑 lsp-utils',
+      label: '🏭 lsp-factory.js',
```
Comment (CJ42) [inline sidebars.js:187]: Maybe we should also mention deprecated in the sidebar menu?
Or put it under a submenu?

```suggestion
      label: '🏭 lsp-factory.js (deprecated)',
```

**Review by CJ42 — COMMENTED** [review]
Nice clean up! Added some review comments

File: docs/learn/expert-guides/universal-profile/create-profile.md (hunk 2 above)
Comment (Hugoo) [reply docs/learn/expert-guides/universal-profile/create-profile.md:69]: You are right, then let's remove this guide and redirect to the one with LSP23 instead.

Comment (CJ42) [reply docs/learn/expert-guides/universal-profile/create-profile.md:69]: Maybe we can comment it out in the sidebar for now until we have a proper strategy for it.

**Review by CJ42 — APPROVED** [review]

**Pull request merged**